postgresql/installingpostgresqlonibmi.md

```diff
@@ -0,0 +1,167 @@
+# Install and configure Postgresql DB Server on IBM i 
+
+Install postgres yum packages from IBM i ACS Open Source Package Management
```

Comment:
@richardschoen 

For this section we should probably only mention install `postgresql12-server` This will pull in `postgresql12` client package and other dependencies e.g. 1postgresql12-libpq1. This should be all that is needed for this install guide.

```
=========================================================================================================================
 Package                              Arch                   Version                  Repository                    Size
=========================================================================================================================
Installing:
 postgresql12-server                  ppc64                  12.2-2                   Artifactory                   26 M
Installing for dependencies:
 postgresql12                         ppc64                  12.2-2                   Artifactory                  2.7 M
 postgresql12-libpq                   ppc64                  12.2-2                   Artifactory                  490 k

Transaction Summary
=========================================================================================================================
Install       3 Packages


```
`postgresql12-contrib` contains various extension modules that may be wanted. This should be considered optional.

`postgresql12-libpgtypes` and `postgresql12-libecpg` are really for development purposes and will be pulled in with the `postgresql12-devel` package.